worlds/landstalker/archipelago.json

```diff
@@ -0,0 +1,6 @@
+{
+  "game": "Landstalker - The Treasures of King Nole",
+  "authors": ["Dinopony"],
+  "minimum_ap_version": "0.6.4",
+  "world_version": "1.8.7"
+}
```

Comment:
```suggestion
}

```

Reply:
Was it agreed that they should have a trailing newline? Currently core game manifests are close to 50/50 on it.